docs/cadc/cadc.rst

```diff
@@ -0,0 +1,564 @@
+.. doctest-skip-all
+
+.. _astroquery.cadc:
+
+*****************************
+Cadc (`astroquery.cadc`)
+*****************************
+
+The Canadian Astronomy Data Centre (CADC) is a world-wide distribution centre for
+astronomical data obtained from telescopes. The CADC specializes in data mining,
+data processing, data distribution and data transferring of very large
+astronomical datasets.
+
+This package allows the access to the data at the CADC
+(http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca)
+
+============
+Basic Access
+============
+
+The most basic ways to access the CADC data and metadata is by region or by
+name. The following example queries CADC for Canada France Hawaii Telescope
+data for a given region and resolves the URLs for downloading the corresponding
+data.
+
+.. code-block:: python
+
+  >>> from astroquery.cadc import Cadc
+  >>>
+  >>> warnings.filterwarnings('ignore')  # turn off astropy warnings (optional)
```

Comment:
specify the module name so you're only filtering astropy warnings; it's a good teaching opportunity

Reply:
Done